### PR TITLE
bcda-3837 Feature: update priority aco ids

### PR DESCRIPTION
**Pull request by SMLuthi**
### Fixes [BCDA-3837](https://jira.cms.gov/browse/BCDA-3837)

The `PRIORITY_ACO_IDS` env var may need to be updated repeatedly to add new priority test aco ids.

### Proposed Changes

Replace the priority aco id env var list with a regex pattern that will match all future priority aco ids (for smoke tests).

### Change Details

- Remove all instances of `PRIORITY_ACO_ID` env var.
- Add `PRIORITY_ACO_REG_EX` env var that contains a regex pattern.
- Replace list search for aco ids with regex built from a pattern in the service.go file 

Sister PR in bcda-ops: https://github.com/CMSgov/bcda-ops/pull/629

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [ ] no PHI/PII is affected by this change

### Acceptance Validation

We can prioritize processing V994 and E9994 requests.

Heres what the regex validates:
image

### Feedback Requested

The usual: Let me know if my GO is up to par.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ services:
       - SSAS_PUBLIC_URL=${SSAS_PUBLIC_URL}
       - SSAS_TIMEOUT_MS=2000
       - BCDA_ENABLE_NEW_GROUP=true
-      - PRIORITY_ACO_IDS=A9990,A9991,A9992,A9993,A9994
+      - PRIORITY_ACO_REG_EX=([A-Z]99([0-8]|9[0-8])$$) # $$ signifies a literal $ in the string
```
Comment (msnook) [inline docker-compose.yml:62]: It is not clear to me why this regex contains `$$` but the regex env var defined in `bcda-ops` contains `$`.

Comment (mtrang1263) [inline docker-compose.yml:62]: Should we also in include `^` at the beginning? And also include the possibility of [A-Z]999/9999

Comment (SMLuthi) [reply docker-compose.yml:62]: Hey Matt, if I keep it as a single `$` we get this Error `ERROR: Invalid interpolation format for "environment" option in service "api": "PRIORITY_ACO_REG_EX=([A-Z]99([0-8]|9[0-8])$)"`. At the VERY bottom of this doc https://github.com/compose-spec/compose-spec/blob/master/spec.md it talks about using double dollar signs when requiring a literal dollar sign.

Comment (SMLuthi) [reply docker-compose.yml:62]: The carrot shouldn't affect anything UNLESS we ever think we are going to get bad id names that start with garbage before the initial [A-Z]. We DID talk about the quadruple 9 though and I need to update that.

Comment (msnook) [reply docker-compose.yml:62]: Ahh, cool. Thanks for the link to the reference doc.

Comment (SMLuthi) [reply docker-compose.yml]: Updated and included a picture from the regex to show what it validates currently.

**Review by mtrang1263 — APPROVED** [review]
LGTM!

**Pull request merged**